src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -263,6 +272,27 @@ else if (participantDataId != null) {
                         }
                         return new Result(200, new GsonBuilder().serializeNulls().create().toJson(map));
                     }
+                    else if (Patch.DDP_PARTICIPANT_ID.equals(patch.getParent())) {
+                        //new additional value for pt which is not in ddp_participant and ddp_participant_record table yet
+                        DDPInstance ddpInstance = DDPInstance.getDDPInstance(patch.getRealm());
+                        String participantId = inTransaction(conn -> {
+                            String newParticipantId = MedicalRecordUtil.writeParticipantIntoDB(conn, patch.getParentId(), ddpInstance.getDdpInstanceId(), 0, "", patch.getUser());
```

Comment:
I think it is good time to introduce `DDParticipantDao` and move `writeParticipantIntoDB` method there to avoid DB connection and interaction here.

Reply:
@gmakharat can you please make that change, thank you